sqlalchemylogger/filters.py

```diff
@@ -0,0 +1,25 @@
+import re
+import logging
+
+
+class ContainsExpression(logging.Filter):
+    """
+    Returns True if the regex is matched in the log message
+    """
+    def __init__(self, regex):
+        self.regex = regex
+
+    def filter(self, record):
+        return bool(re.search(self.regex, record.msg))
+
+
+class DoesNotContainExpression(logging.Filter):
+    """
+    Returns True if the regex is NOT matched in the log message
+    """
+    def __init__(self, regex):
+        self.regex = regex
```

Comment:
You could compile de regex:
```
self.regex = re.compile(regex)
```

And, then, use it like that:
```
self.regex.search(record.msg)
```

Reply:
good idea, thanks

Reply:
fixed [here](https://github.com/danduk82/poc_oereb_logger/pull/4/commits/e6b198f9257f46965e31d6cb535e389ab68bc1e4)